package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.8"
+    "nan": "2.14.0"
```

Comment:
Is there a reason to depend on a specific nan version instead of `^2.14.0`? Nan follows semver, releases are backwards compatible.

Reply:
precise versions guarantee what we're getting, but that sounds reasonable for `nan`, doing `^2.14.0` or `2.14`

Reply:
we could